web_ui/ui.go

```diff
@@ -557,6 +558,7 @@ func GetEngine() (*gin.Engine, error) {
 	gin.SetMode(gin.ReleaseMode)
 	engine := gin.New()
 	engine.Use(gin.Recovery())
+	engine.Use(gzip.Gzip(gzip.DefaultCompression, gzip.WithExcludedPaths([]string{"/api/v1.0/prometheus"})))
```

Comment:
When I was testing, the `/metrics` endpoint was zipped into a file, which is not expected. This endpoint should be added to the excluded list as well